atomic_reactor/inner.py

```diff
@@ -562,6 +562,12 @@ def build_docker_image(self):
 
             signal.signal(signal.SIGTERM, signal.SIG_DFL)
 
+    def add_release_var(self, build_id):
+        dockerfile = df_parser(self.builder.df_path, workflow=self)
+        if self.source.config.release_env_var:
+            release_line = "ENV {}={}".format(self.source.config.release_env_var, build_id)
+            dockerfile.add_lines(release_line, all_stages=True, at_start=True)
```

Comment:
Wondering about 'all_stages'. I think it makes sense to add this to all stages, but @rcerven I think you'd originally planned to add this only to the last one. Have a preference?

Reply:
mmm I mean we can add it to all stages, that doesn't hurt

Reply:
I removed all_stages.